addproduct.php

```diff
@@ -0,0 +1,121 @@
+<?php
+
+include('db.php');
+session_start();
+if(!isset($_SESSION['user'])){
+    header('Location: login.php');
+}
+
+$pname = "";
+$pdetail = "";
+$price = "";
+$errors = array(
+    'pname' => '',
+    'price' => '',
+    'file' => ''
+);
+if (isset($_POST['submit'])) {
+    $pname =  mysqli_real_escape_string($conn, $_POST['pname']);
+    $pdetail = mysqli_real_escape_string($conn, $_POST['pdetail']);
+    $price = mysqli_real_escape_string($conn, $_POST['price']);
+
+    if (empty($pname)) {
+        $errors['pname'] = "Please enter the product name";
+    }
+    if (empty($price)) {
+        $errors['price'] = "Please enter the price";
+    }
+    $new_path = "img";
+        $allowedfile = ['jpg', 'png', 'jpeg'];
```

Comment:
You can write this as a separate function, since it is used in both add and edit pages.
While doing development, it is a best practice to avoid repeated codes.